dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -60,12 +56,10 @@
   private static final HelpFormatter helpFormatter = new HelpFormatter();
 
   private final CommandLine commandLine;
-  private ImmutableList<Path> cachedInputClasspath;
```

Comment:
LinkageCheckerArguments is no longer responsible to resolve the class path. Therefore it does not know class path (`cachedInputClasspath` variable).